src/cartridge_info.c

```diff
@@ -0,0 +1,111 @@
+/*
+ * cartridge_info.c - cartridge information
+ *
+ * Copyright (C) 2001-2010 Piotr Fusik
+ * Copyright (C) 2001-2010 Atari800 development team (see DOC/CREDITS)
```

Comment:
Minor nit: you could use 2020 as the last year, as you are creating a new file.

Reply:
This looks like being forgotten to update.